hack/lib/install.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 The KubeEdge Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# check if kubectl installed
+function check-prerequisites {
+  echo "checking prerequisites"
+  which kubectl >/dev/null 2>&1
+  if [[ $? -ne 0 ]]; then
+    echo "kubectl not installed, exiting."
+    exit 1
+  else
+    echo -n "found kubectl, " && kubectl version --short --client
+  fi
+}
+
+# check if kind installed
+function check-kind {
+  echo "checking kind"
+  which kind >/dev/null 2>&1
+  if [[ $? -ne 0 ]]; then
+    echo "installing kind ."
+    GO111MODULE="on" go get sigs.k8s.io/kind@v0.5.1
+    export PATH=$PATH:$GOPATH/bin
```

Comment:
I think this step should be one of the prerequisites. It may add GOPATH/bin to the PATH twice

Reply:
It only try add $GOPAHT/bing to $PATH when kind is newly installed, doesn't really matter if added twice.